restylers/gn/Dockerfile

```diff
@@ -1,16 +1,9 @@
-FROM ubuntu:20.04
+FROM frolvlad/alpine-glibc:alpine-3.12
```

Comment:
I think I'm being paranoid, but I do raise my eyebrows a bit at moving to a "random" base image here that installs its own compiler, and what that might mean for the resulting image that would be running with access to my user's private source code...

Is there anything you can say to help quell my fears? Would it be terrible to say on (perhaps not ubuntu, but) another more official glibc-containing image?

Reply:
Done.

Reply:
Oh wow, thanks for so quickly addressing this. I was expecting to have a bit of discussion and push-back :smile:

Reply:
This is actually an earlier version of the patch. I thought it shouldn't be more than 10M, which led down to trying to find a container with only glibc. That alpine one is the closest thing I could find.

In practice it really is <10M anyway, because the ubuntu 20.04 is a sunk cost for restyler-clang-format.